policyengine_us/reforms/reconciliation/ctc_ssn.py

```diff
@@ -28,14 +28,26 @@ class filer_meets_ctc_identification_requirements(Variable):
         def formula(tax_unit, period, parameters):
             # Both head and spouse in the tax unit must have valid SSN card type to be eligible for the CTC
             person = tax_unit.members
-            is_head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+            head_or_spouse = person("is_tax_unit_head_or_spouse", period)
             eligible_ssn_card_type = person(
                 "meets_ctc_identification_requirements", period
             )
             ineligible_head_or_spouse = (
```

Comment:
no need to check joint status
```suggestion
            p = parameters(period).gov.contrib.reconciliation.ctc
            if p.one_person_ssn_req:
                return tax_unit.any(head_or_spouse & eligible_ssn_card_type)
            return ~tax_unit.any(head_or_spouse & ~eligible_ssn_card_type)
```